packages/airnode-admin/test/e2e/cli.feature.ts

```diff
@@ -50,7 +50,14 @@ describe('CLI', () => {
       return execSync(`node ${CLI_EXECUTABLE} ${formattedCommand}`).toString().trim();
     } catch (e: any) {
       // rethrow the output of the CLI
-      throw new Error(e.stdout.toString().trim());
+      const error = e.stdout.toString().trim();
+      // Checking for ethers errors and throwing a custom error message here because
+      // the ethers error will contain transaction information which is not constant,
+      if (error.includes('eip-1559 transaction do not support gasPrice')) {
```

Comment:
I don't think you need this. Afaik, jest .toThrow(str) only check if `str` is a substring of the error message.

Reply:
That works, thanks!